src/pretalx/submission/models/resource.py

```diff
@@ -40,6 +41,13 @@ class Resource(PretalxModel):
     is_public = models.BooleanField(
         default=True, verbose_name=_("Publicly visible resource")
     )
+    hide_until_event_day = models.BooleanField(
```

Comment:
I don't think a new field is the right choice here, in terms of data-modeling. What would it mean for a resource to be set to `is_public=False` but `hide_until_event_day=True`? Would the user expect that the resource would be hidden from the organisers until the event day?